From b7974c768887aa3d0ec608d5649fbcb415060e3e Mon Sep 17 00:00:00 2001
From: Heba Aamer <heba93aamer@gmail.com>
Date: Tue, 27 Jan 2015 22:35:30 +0200
Subject: [PATCH 163/193] staging: fbtft: fix spaces required around that '='

This patch fixes the following checkpatch.pl error:
fix spaces required around that '='

Signed-off-by: Heba Aamer <heba93aamer@gmail.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
---
 drivers/staging/fbtft/fb_pcd8544.c | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/drivers/staging/fbtft/fb_pcd8544.c b/drivers/staging/fbtft/fb_pcd8544.c
index a5f0ad4..e44eecb 100644
--- a/drivers/staging/fbtft/fb_pcd8544.c
+++ b/drivers/staging/fbtft/fb_pcd8544.c
@@ -117,10 +117,10 @@ static int write_vmem(struct fbtft_par *par, size_t offset, size_t len)
 
 	fbtft_par_dbg(DEBUG_WRITE_VMEM, par, "%s()\n", __func__);
 
-	for (x=0; x<84; x++) {
-		for (y=0; y<6; y++) {
+	for (x = 0; x<84; x++) {
+		for (y = 0; y<6; y++) {
 			*buf = 0x00;
-			for (i=0; i<8; i++) {
+			for (i = 0; i<8; i++) {
 				*buf |= (vmem16[(y*8+i)*84+x] ? 1 : 0) << i;
 			}
 			buf++;
-- 
2.1.4

